integration/app_integration_test.go

```diff
@@ -238,6 +239,8 @@ func setup(t *testing.T) *pack {
 	tr := utils.NewTracer(utils.ThisFunction()).Start()
 	defer tr.Stop()
 
+	log := testlog.FailureOnly(t)
```

Comment:
Does this mean integration tests are not going to output info/warn messages anymore, or is there a way to toggle them on? Just asking cause as I was working on an integration tests for db access, I used info outputs quite a bit to troubleshoot issues.

Reply:
No, the test output will be streamed as expected in verbose mode.